dockerfiles/nexus3/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM snyk/ubuntu as base
+
+MAINTAINER Snyk Ltd
+
+USER root
+
+RUN apt update && apt install -y make python gcc
+
+RUN apt-get update && apt-get install -y ca-certificates
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+
+ENV PATH=$PATH:/home/node/.npm-global/bin
+
+RUN npm install --global snyk-broker
+
+
+
+FROM snyk/ubuntu
+
+ENV PATH=$PATH:/home/node/.npm-global/bin
+
+COPY --from=base /home/node/.npm-global /home/node/.npm-global
+
+COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+# Don't run as root
+WORKDIR /home/node
+USER node
+
+# Generate default accept filter
+RUN broker init nexus3
+
+######################################
+# Custom Broker Client configuration #
+# Redefine in derived Dockerfile,    #
+# or provide as runtime args `-e`    #
+######################################
+
+# Your unique Broker identifier
+# ENV BROKER_TOKEN <broker-token>
+
+# The URL to your Nexus Repository Manager
+# NOTA BENE: without `/` character in the end of URL
+# ENV NEXUS_URL=<username>:<password>@<your.nexus.hostname>/repository
```

Comment:
not quite correct - appreciate this was how it was before anyway

Reply:
this is the URL for nexus3. what do you mean?

Reply:
its missing `https://` etc. Theres probably other missing stuff in these comments. Not a major issue, can look over it